fe/fe-core/src/main/java/org/apache/doris/load/routineload/RoutineLoadManager.java

```diff
@@ -426,7 +426,10 @@ public long getAvailableBeForTask(long previoudBeId, String clusterName) throws
             // 1. Find if the given BE id has available slots
             if (previoudBeId != -1L) {
                 int idleTaskNum = 0;
-                if (beIdToConcurrentTasks.containsKey(previoudBeId)) {
+                if (beIdToMaxConcurrentTasks.containsKey(previoudBeId)) {
+                    idleTaskNum = 0;
+                    LOG.warn("backend:" + previoudBeId + " not found, cannot get idleTaskNum.");
```

Comment:
No need to print warn log here because this is not a root cause.
The root cause is because BE is down. And `beIdToMaxConcurrentTasks` only contains alive Backends.
So here we can just check if `beIdToMaxConcurrentTasks.containsKey(previoudBeId)`

Reply:
And BE crash because of this #6716. Need further debug.